packages/vector_graphics_compiler/test/parsers_test.dart

```diff
@@ -172,6 +172,23 @@ void main() {
     expect(parseDoubleWithUnits('1pt', theme: const SvgTheme()), 1 + 1 / 3);
   });
 
+  test('Parse SVG with "none" value', () {
+    final TestColorMapper mapper = TestColorMapper();
+    final SvgParser parser = SvgParser(
+      '<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg"><rect x="100" y="10" width="80" height="80" fill="red" stroke-width="none" /></svg>',
```

Comment:
What I'm reading from SVG spec here https://svgwg.org/svg2-draft/geometry.html#Sizing is that width/height are treated as CSS properties, and in CSS i believe none clears a previously established default from the cascade. Its not meaningful for SVGs parsed by this library since we don't actually support CSS, but given that folks drop SVGs from external systems in here I think its reasonable to 'support' it by being tolerant of the value.

Reply:
So should `none` have special handling? That would be a very different fix from what the current code in the PR is doing.